assets/src/components/SelectCard.js

```diff
@@ -117,23 +136,54 @@ const SelectCard = props => {
 
   function getCardImage (cardData) {
     if (cardData && cardData.image) {
-      return <CardMedia className={classes.media} image={cardData.image} title={cardData.title} />
+      return (
+        <>
+          <Link className={classes.viewLink} tabIndex={-1} to={cardData.path}>
+            <CardMedia className={classes.media} image={cardData.image} title={cardData.title} />
+          </Link>
+        </>
+      )
     } else {
       return null
     }
   }
 
+  function getHelpLink (cardData) {
+    if (cardData.helpUrl) {
+      return (
+        <Typography gutterBottom variant='h5' component='h4' className={classes.title}>
```

Comment:
Nit-pick: Is `Typography` necessary here?

Reply:
It is allowing the icon to align with the title text.  Would there be a better way to do it?